src/components/pages/Preview/index.tsx

```diff
@@ -24,7 +24,7 @@ const PreviewPage: React.FC<Props> = () => {
     <ThemeProvider>
       <DndProvider>
         <AuthenticationRequiredPage>
-          <CanvasArea isBuilt />
+          <CanvasArea isBuilt isInEditor={false} />
```

Comment:
Just don't pass isInEditor and by handling it in the CanvasArea like `!!isInEditor` it'll be false

Reply:
Updated. And also set `<CanvasArea inEditor />` in `page/EarthEditor`.